### ISSUE TITLE
Sometimes the app doesn't load

### ISSUE THREAD

**Issue by brandonhaslegs** [kawamurakazushi/figma-map-maker#42]
Not exactly sure what's happening here.

![image](https://user-images.githubusercontent.com/4406983/150169744-c7294893-f06a-44cf-9d55-1a786c6c2791.png)

**Comment by nseifelnasr** [comment]
Yeah, I've been getting the same thing, not sure what's going on tbh

**Comment by brandonhaslegs** [comment]
Yeah been like that for days. The maps never loads anymore.

**Comment by lutfiblog** [comment]
me too bro, i just install map maker plugin

**Comment by balisolutions** [comment]
Same here, not working yet

**Comment by jon-tet** [comment]
Same
